### PR TITLE
Reducers and actions

### PR DESCRIPTION
**Pull request by iLynette**
In this project i implemented the following;

- [ ] Write your book's actions and reducer (in /src/redux/books/books.js) using the [ducks pattern](https://github.com/erikras/ducks-modular-redux).

- [ ] Define action types for adding and removing a book.

- [ ] Set the initial state to be an empty array of books.

- [ ] Export Action Creators for your actions.

- [ ] Write your reducer and export it as default.

- [ ] Define state changes for the actions that you created.

- [ ] In case of unknown action - return the current state.

- [ ] Write your categories actions and reducer (in /src/redux/categories/categories.js) using the [ducks pattern](https://github.com/erikras/ducks-modular-redux).

- [ ] Define an action type for checking the status.

- [ ] Set the initial state to be an empty array of categories.

- [ ] Export Action Creators for your actions.

- [ ] Write your reducer and export it as default.

- [ ] For the check status action return a string "Under construction".

- [ ] In case of unknown action - return the current state.

Configure the Redux Store (/src/redux/configureStore.js):

- [ ] Importing the necessary methods from Redux.

- [ ] Combine both reducers into a root reducer by using the combineReducers function.

- [ ] Use the root reducer to create your store.

### CODE REVIEW

File: src/redux/configureStore.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { combineReducers, createStore } from 'redux';
+import booksReducer from './books/books';
+
+const reducer = combineReducers({
+  booksReducer,
+});
```
Comment (omar-labana) [inline src/redux/configureStore.js]: - [ ] Kindly, import and combine the `categoriesReducer` as instructed in the requirements.

**Review by omar-labana — CHANGES_REQUESTED** [review]
Hi @iLynette, 

Excellent work so far! 👌
You have implemented almost all the required features without any errors or bugs. ✔️ 
There are some cases that you need to adjust but you are almost done! 😀

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/configureStore.js (hunk 1 above)
Comment (iLynette) [reply src/redux/configureStore.js]: Thank you for the review @omar-labana I have resolved it.

**Review by uwadonat — APPROVED** [review]
Hi **@[iLynette](https://github.com/iLynette),** 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**